.travis.yml

```diff
@@ -1,18 +1,29 @@
 language: julia
+
 os:
   - linux
   - osx
+
 julia:
   - 0.7
   - 1.0
   - nightly
+
 matrix:
   allow_failures:
     - julia: nightly
-# script:
-#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("TSVD"); Pkg.test("TSVD"; coverage=true)'
+
 after_success:
-  - julia -e 'cd(Pkg.dir("TSVD")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())'
-  - julia -e 'Pkg.add("Documenter")'
-  - julia -e 'cd(Pkg.dir("TSVD")); include(joinpath("docs", "make.jl"))'
+  - julia --project=coverage/ -e 'using Pkg; Pkg.instantiate()'
+  - julia --project=coverage/ coverage/coverage.jl
+
+jobs:
+  include:
+    - stage: "Deploy docs"
+      julia: 1.0
+      os: linux
+      script:
+        - julia --project=docs/ -e 'using Pkg; Pkg.instantiate()'
+        - julia --project=docs/ -e 'using Pkg; Pkg.add(PackageSpec(path=pwd()))'
```

Comment:
If you do this you don't actually need to check in the `Manifest.toml`, since that will lock the version of e.g. Documenter etc.